scripts/create_sample_worksheet.py

```diff
@@ -87,6 +87,7 @@ def create(self):
         print('Done.')
 
     def test_print(self):
+        self._wait_for_bundles_to_finish()
```

Comment:
Why do we have to wait for bundles to finish? I think in the bundle regex comparison,`staging` is acceptable too.

Reply:
I guess it's because of these lines:
https://github.com/codalab/codalab-worksheets/blob/e47d18baef92a97a38565c2e363e8421c9ac39fc/scripts/create_sample_worksheet.py#L359-L361
When it explicitly looks for bundles in `ready`, if any of those valid bundles are not in `ready` state, when the program compares the output against the exepcted output, it will find the differences.